extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -334,8 +295,7 @@ export const AzureProvisionDialog: React.FC = () => {
   const [importConfig, setImportConfig] = useState<any>();
 
   const [page, setPage] = useState(PageTypes.ConfigProvision);
-  const [group, setGroup] = useState<IGroup[]>();
-  const [listItems, setListItem] = useState<(ResourcesItem & { name; icon })[]>();
+  const [listItems, setListItem] = useState<(ResourcesItem & { icon?: string })[]>();
```

Comment:
do you want the default value be undefined or an empty list?

Reply:
I'm removing the name from the type as it isn't needed anywhere.  I'm not changing the default value.